### PR TITLE
ui: Include nspace in up/downstream link when nspaces are enabled

### PR DESCRIPTION
**Pull request by johncowen**

### CODE REVIEW

**Comment by rboyer** [comment]
Remember this will have to be separately cherrypicked to `1.9.0` in addition to automatically to `1.9.x`

File: ui/packages/consul-ui/app/components/topology-metrics/card.hbs [hunk 1]
```diff
@@ -1,5 +1,9 @@
 <a class="card"
-  href={{href-to "dc.services.show.index" @item.Name}}
+  href={{if
+   (and (env 'CONSUL_NSPACES_ENABLED') (not-eq @item.Namespace @service.Namespace))
```
Comment (mikemorris) [inline ui/packages/consul-ui/app/components/topology-metrics/card.hbs:3]: This template language is a bit difficult for me to parse - does this also cover the `else` cases properly for:
- [ ] namespaces disabled = regular service link
- [ ] namespaces enabled, service in same namespace = link to appropriately namespaced service?

Comment (johncowen) [reply ui/packages/consul-ui/app/components/topology-metrics/card.hbs:3]: 👍 Yeah it'll cover both those cases.

**Review by mikemorris — APPROVED** [review]

**Comment by johncowen** [comment]
Also manually picked this onto `release/1.9.0`

**Pull request merged**